valohai_yaml/objs/step.py

```diff
@@ -106,13 +106,23 @@ def get_parameter_defaults(self, include_flags: bool = True) -> Dict[str, Any]:
             if parameter.default is not None and (include_flags or parameter.type != 'flag')
         }
 
-    def get_input_defaults(self) -> Dict[str, List[str]]:
+    def get_serialized_parameters(self, include_flags: bool = True) -> Dict[str, Any]:
+        """Get a dict mapping parameter names to their defaults (if set)."""
```

Comment:
This seems copy-pasted from the other implementation, and as such isn't really correct (since this implementation will return a `''` for non-set defaults.